src/main.cpp

```diff
@@ -70,6 +70,12 @@ class Konstructs: public nanogui::Screen {
         menu_state(0) {
 
         using namespace nanogui;
+
+        printf("OpenGL vendor: %s\n", glGetString(GL_VENDOR));
```

Comment:
Maybe it could be moved to first thing in the main function, or does it require Open GL to be initialized?